examples/layouts/flex/lv_example_flex_1.c

```diff
@@ -28,7 +28,7 @@ void lv_example_flex_1(void)
         lv_obj_set_size(obj, 100, LV_PCT(100));
 
         label = lv_label_create(obj);
-        lv_label_set_text_fmt(label, "Item: %u", i);
+        lv_label_set_text_fmt(label, "Item: %lu", i);
```

Comment:
These should probably just be replaced with the `LV_PRI` macros as well; there is no way to easily get the old-style placeholders to work correctly on every platform.

Reply:
Thanks @embeddedt, I will look into that. I see some of the checks for certain platforms did fail so you are right about using the `LV_PRI` macros.